<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="regButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>70</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>42</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff5500;&quot;&gt;注册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="msg_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>291</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt;昵称:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt;密码:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt;确认密码:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="nick_nameLineEdit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="maxLength">
         <number>32763</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwdLineEdit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="maxLength">
         <number>32763</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="repPasswdLineEdit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="maxLength">
         <number>32763</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>178</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;性别:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="male_radioButton">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>男</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="female_radioButton">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 女</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nick_nameLineEdit</tabstop>
  <tabstop>passwdLineEdit</tabstop>
  <tabstop>repPasswdLineEdit</tabstop>
  <tabstop>male_radioButton</tabstop>
  <tabstop>female_radioButton</tabstop>
  <tabstop>regButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
